### ISSUE TITLE
Door Man - Events - View list of events

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#136]
_depends on #135_
## Background
An Event Manager will have the ability to view a list of all events they have to manage. This can be accessed from the account page.

## Scenario: An event Manager selects Manage Events from their Account Screen
- [ ] When I select `Manage Events`
- [ ] Then I am directed to a list of events that I'm managing
- [ ] And I see the Name/Location/Date of Event 

Mock: 
![screen shot 2018-10-18 at 2 01 28 pm](https://user-images.githubusercontent.com/14582709/47177569-5bd41b00-d2de-11e8-902f-99b6e73e389e.png)

**Comment by brittanygay** [comment]
Working on the styling for this screen next

**Comment by brittanygay** [comment]
**Update**
There's a React Native Truncation bug that I keep getting on this screen. The second line can't wrap, because then the card's height will be lengthened, and trying to truncate the details also isn't working due to the bug I came across. It would be better if we could change up the design a bit and maybe eliminate some details from the bottom, maybe keeping the date or the location (but maybe not both)? 

I'll continue to research a solution for the issue, but here are screenshots of what's happening when i try to truncate the second line (the second line basically won't truncate).

<img width="349" alt="screen shot 2018-10-22 at 11 46 21 am" src="https://user-images.githubusercontent.com/14582709/47306612-8a116d80-d5f2-11e8-95d9-cf461a2b23de.png">

View if we only had to truncate the artist name (which works, but still need to clean up some alignment issues):
<img width="320" alt="screen shot 2018-10-22 at 12 03 13 pm" src="https://user-images.githubusercontent.com/14582709/47306634-9b5a7a00-d5f2-11e8-9729-cfd998019d30.png">
